client-go/applyconfiguration/internal/internal.go

```diff
@@ -0,0 +1,58 @@
+/*
```

Comment:
any idea what is this applyconfiguration pkg about?

Reply:
looks like a way of type-safe doing server-side applies https://pkg.go.dev/k8s.io/client-go/applyconfigurations